README.md

```diff
@@ -19,13 +19,13 @@
 
 ## About
 
-Scratch Addons is a WebExtension (supports both Chrome and Firefox). Scratch Addons' mission is to combine all existing Scratch extensions, userscripts and userstyles into a single easy-to-access place, while still letting users choose which ones to enable.
+Scratch Addons is a WebExtension supporting both Chrome and Firefox. Scratch Addons' mission is to combine all existing Scratch extensions, userscripts, and userstyles into a single easy-to-access place, while still letting users choose which ones to enable.
 
 ### What's actually an "addon"?
 
-An addon is similar to an extension or a userscript, but they use special APIs provided by the Scratch Addons extension. These APIs allow addons to run scripts on a Scratch page (userscripts), run scripts on the background (persistent scripts), or apply styles to the Scratch website (userstyles).
+An addon is similar to an extension or a userscript, but they use special APIs provided by the Scratch Addons extension. These APIs allow addons to run scripts on a Scratch page (userscripts) or apply styles to the Scratch website (userstyles).
 
-Userscripts and persistent scripts can use the `addon.*` JavaScript APIs, which allow them to obtain Scratch-related information (for example, get the current logged in user) and also use extension APIs (like sending notifications).
+Userscripts can use the `addon.*` JavaScript APIs, which allow them to obtain Scratch-related information (for example, get the currently logged-in user) and also use extension APIs (like sending notifications).
```

Comment:
Don't forget to update FAQ

Reply:
FAQ has an open pr to make it less techy

Reply:
Those long README are those that don't have wiki or other types of documentation (see youtube-dl). We have exaustive amount of documentation but we put it on the README. If README can go simpler than I rather. 

The README could be also used for advertisement like the website.